app/colors/default.css

```diff
@@ -245,12 +245,11 @@ webview {pointer-events: none;position: fixed;display: none;}
 #cookiespage .remove {height: 1em;width: 1em;margin: auto 0;cursor: pointer;}
 /* downloads */
 #downloadspage .download {background: var(--special-page-element-bg);margin-bottom: 1em;border: .1em solid var(--special-page-element-border);}
-#downloadspage .title {margin: .4em;font-size: 1.5em;width: calc(100% - 4em);}
+#downloadspage .title {margin: .4em;font-size: 1.5em;width: calc(100% - 4em);cursor:pointer;}
```

Comment:
Where is this used? On both the notification and download page has a pointer cursor due to the filelocation class.

Reply:
This style is for the download *title*, which does not have the filelocation class.

Reply:
Right, now it makes a bit more sense, thanks for clearing that up.

Reply:
Also, only the title of completed downloads should be clickable (and thus have a pointer cursor).

Reply:
Opening an in-progress download can be useful as well, e.g. videos can be played while they're still downloading.

Reply:
Yes, for in progress downloads this would be nice, but I think the filename is different while downloading, and not all files can be opened that way. Finally it should be disabled for failed downloads regardless.